.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,22 +13,10 @@ GitHub Issue: #
 <!-- (Please describe the changes that this PR introduces.) -->
 
 
-## Impact on version
```

Comment:
Keep this section, but add some more explanation:

## Impact on version
What would be the impact on the version if these changes would be merged back into the main repository?

Reply:
done